gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -35,10 +35,10 @@
 public abstract class GoogleCloudStorageOptions {
 
   /** Default setting for enabling use of GCS gRPC API. */
-  public static final boolean ENABLE_GRPC_DEFAULT = false;
+  public static final boolean DEFAULT_GRPC_ENABLED = false;
 
   /** Default setting for enabling use of the Traffic Director for GCS gRPC API. */
-  public static final boolean ENABLE_TRAFFIC_DIRECTOR_DEFAULT = true;
+  public static final boolean DEFAULT_TRAFFIC_DIRECTOR_ENABLED = true;
 
   /** Default setting to prefer DirectPath for gRPC. */
   public static final boolean DIRECT_PATH_PREFERRED_DEFAULT = true;
```

Comment:
Seems like we need to rename all default constants to have `DEFAULT_` prefix

Reply:
Done. Thanks!